test/test_matmul_cuda.py

```diff
@@ -367,11 +367,8 @@ def _test_tautological_mm(self, device: str = "cuda",
         (out_fp8, amax_fp8) = torch._scaled_mm(x_fp8, y_fp8, out_dtype=out_dtype)
         if out_dtype is not None:
             self.assertEqual(out_dtype, out_fp8.dtype)
-        if out_dtype not in [torch.float16, torch.bfloat16, torch.float]:
```

Comment:
You made a lot of ROCm specific changes but this change applies to CUDA as well. Are we sure this doesn't break something on their end?